README.md

```diff
@@ -1,6 +1,28 @@
-# Hyperchains Contract
+Hyperchains Contract
+====================
+
+Proxy lib for the AEternity project. When used as a dependency, provides compiled and ready to use StakingContract.json
+
+The source code (in Sophia programming language) for the contract is in `/priv/SimpleElection.aes`
+
+Build
+-----
+
```

Comment:
Say something like:
```suggestion
Contract can be build and tested as below (see "Develop locally") or included in an rebar project. 
```

Reply:
I mean, previously was only one way to use this repo. There was the "aeproject" javascript stuff. Now you kind of add few files to make it possible to include this as a rebar project to Erlang code. Readme should reflect that nicely. 

There are instructions to compile work with it using npm below. Change the readme so the whole text "makes sense"